src/components/checkbox/Checkbox.tsx

```diff
@@ -65,32 +88,35 @@ export function Checkbox(props: Readonly<Props>) {
           id={id}
           onCheckedChange={handleChange}
           onFocus={onFocus}
-          title={title}>
+          title={title}
+          {...(hasError && checked === false ? { 'data-error': true } : {})}>
```

Comment:
if it's in error, you can still check it, though - does that make sense?

Reply:
Following the conversation with Marcio in Figma, the use case for error is mainly when you must check it for license purposes or that kind of thing.
So it doesn't seem to make sense to allow an error state for checked checkbox, hence the condition.
I'll add comment.